ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/RequestHandler.java

```diff
@@ -76,14 +83,22 @@ public RequestHandler(
 
   public KsqlEntityList execute(
       final List<ParsedStatement> statements,
-      final Map<String, Object> propertyOverrides
+      final Map<String, Object> overrides
+  ) {
+    return execute(statements, overrides, injectorFactory.apply(ksqlEngine, serviceContext));
+  }
+
+  private KsqlEntityList execute(
+      final List<ParsedStatement> statements,
+      final Map<String, Object> propertyOverrides,
```

Comment:
nit: use the same name for overrides/propertyOverrides in both `execute` methods